### PR TITLE
chore: update coin libs

### PR DESCRIPTION
**Pull request by marianogoldman**
## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

File: packages/platform-sdk-egld/src/services/transaction.ts [hunk 1]
```diff
@@ -42,18 +52,28 @@ export class TransactionService implements Contracts.TransactionService {
...
+
+		const tx: Transaction = new Transaction(
+			{
+				receiver: Address.fromString(input.data.to),
+				value: Balance.egld(input.data.amount),
+				gasPrice: new GasPrice((input.fee as unknown) as number),
+				gasLimit: new GasLimit((input.feeLimit as unknown) as number),
+				data: new TransactionPayload(input.data.memo),
+			});
+		// tx.setNonce(account.nonce); // TODO Do we need specify this? If so we will need to sync account first, or receive it via param
```
Comment (faustbrian) [inline packages/platform-sdk-egld/src/services/transaction.ts:73]: Unless the library automatically sets the nonce we will need to pass it in.

File: packages/platform-sdk-egld/src/services/transaction.ts [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+import {
+	Address,
+	Balance,
+	GasLimit,
+	GasPrice,
+	Mnemonic,
+	Transaction,
+	TransactionPayload,
+	UserSecretKey
+} from "@elrondnetwork/erdjs/out";
```
[the remark this answers is not in the document]
Comment (marianogoldman) [reply packages/platform-sdk-egld/src/services/transaction.ts:13]: Good question. Checking.

File: packages/platform-sdk-egld/src/services/transaction.ts (hunk 1 above)
Comment (faustbrian) [reply packages/platform-sdk-egld/src/services/transaction.ts:73]: @marianogoldman but we don't need to sync here, it's parameter that can already be passed in.

Comment (marianogoldman) [reply packages/platform-sdk-egld/src/services/transaction.ts:73]: I agree... this is their [example code](https://elrondnetwork.github.io/elrond-sdk-docs/erdjs/latest/#creating-value-transfer-transactions) for signing transaction.

I think we need to pass it in, as I don't think the library has the means (at this stage) to figure that out. And yes, I guess it should come as a param in the `TransactionInputs`.

Comment (marianogoldman) [reply packages/platform-sdk-egld/src/services/transaction.ts:73]: We don't have any tests that actually ever submit a transaction (like we have for Cardano). Has this ever been tested against the network?

Comment (faustbrian) [reply packages/platform-sdk-egld/src/services/transaction.ts]: I've manually tested it with the previous lib.

**Pull request merged**